src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsMoreEmptyValues.java

```diff
@@ -61,13 +36,10 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
         Class<?> returnType = invocation.getMethod().getReturnType();
         return returnValueFor(returnType);
     }
-    
+
     Object returnValueFor(Class<?> type) {
         if (type == String.class) {
             return "";
```

Comment:
Actually I don't think that's part of this PR. And I find that returning empty string instead of `null` more debattable (hence a separate PR).